runtime/doc/lua.txt

```diff
@@ -4551,16 +4551,6 @@ vim.snippet.active({filter})                            *vim.snippet.active()*
     Returns `true` if there's an active snippet in the current buffer,
     applying the given filter if provided.
 
-    You can use this function to navigate a snippet as follows: >lua
-        vim.keymap.set({ 'i', 's' }, '<Tab>', function()
```

Comment:
instead of dropping this , worth updating it with the example from your PR description?

Reply:
oh, there's already an example below at `:help vim.snippet.jump()` 👍